classes/PodsAPI.php

```diff
@@ -155,12 +155,16 @@ public function save_wp_object( $object_type, $data, $meta = array(), $strict =
 			$object_type = 'term';
 		}
 
+		$is_meta_object = in_array( $object_type, array( 'post', 'term', 'user', 'comment' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 5149bb30bfe969a31482016b18c1d9bc</sub>